src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/enumeration/CsipStatus.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.enumeration
+
+enum class CsipStatus {
+  CsipClosed,
+  CsipOpen,
+  AwaitingDecision,
+  AcctSupport,
+  PlanPending,
+  InvestigationPending,
+  NoFurtherAction,
+  SupportOutsideCsip,
+  ReferralSubmitted,
+  ReferralPending,
+  Unknown,
```

Comment:
can we have a string property for these enums for their display string, and the CsipRecord model to return both a status and a statusDisplayName, so that the backend will be fully in charge of those statuses, and the frontend doesn't need to maintain a matching set of enum and its display name mapping?

Reply:
i expect enum value doesn't ever change and can be relied upon for statistics analysis. while display name can be changed for whatever reason.

Reply:
you're right front end just need the actual display name and doesn't need the enum value.